### PR TITLE
Change from .html to .md

### PR DESCRIPTION
**Pull request by RiccardoMasutti**

### CODE REVIEW

File: docs/FAQ/FAQ-Contribution.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ It's not the case anymore.
 Though it's most likely not an issue to accept Bitcoin donations, zkSNACKs team discontinued accepting donations because they don't want to get into accidental legal gray areas.
 Wasabi has a coordinator fee that is capped at 0.3% for every CoinJoin and it became zkSNACKs' main income.
 To help Wasabi development, just CoinJoin your bitcoins via Wasabi!
-If you want to read more about fees here is a nice explanation: [What are the fees for the CoinJoin](https://docs.wasabiwallet.io/FAQ/FAQ-UseWasabi.html#what-are-the-fees-for-the-coinjoin).
+If you want to read more about fees here is a nice explanation: [What are the fees for the CoinJoin](https://docs.wasabiwallet.io/FAQ/FAQ-UseWasabi.md#what-are-the-fees-for-the-coinjoin).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Contribution.md]: Have you tested this?
It was working properly but with this change it doesn't.

File: docs/FAQ/FAQ-Installation.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
 It's always best to download software directly from the official source acknowledged by the developers.
 You can find the recent version of the compiled packages for Linux, Windows and Mac available on the official [wasabiwallet.io](https://wasabiwallet.io).
 In order to preserve your network level privacy from the very first step on, please consider visiting the tor hidden service [wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion](http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion).
-The old versions of the software can be found in the [releases](https://github.com/zksnacks/walletwasabi/releases) of the GitHub repository, [here](https://github.com/zksnacks/walletwasabi) you also find the libre & open source code for when you want to [build it yourself](https://docs.wasabiwallet.io/using-wasabi/BuildSource.html).
+The old versions of the software can be found in the [releases](https://github.com/zksnacks/walletwasabi/releases) of the GitHub repository, [here](https://github.com/zksnacks/walletwasabi) you also find the libre & open source code for when you want to [build it yourself](https://docs.wasabiwallet.io/using-wasabi/BuildSource.md).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Installation.md]: same here

**Comment by yahiheb** [comment]
NACK
I didn't review all but the first two links were working properly and with the proposed change they are not.

**Comment by RiccardoMasutti** [comment]
@yahiheb thanks for the review. A few were absolute links instead of relative and didn't work.

Fixed

**Comment by yahiheb** [comment]
Everything was working properly before the changes. there is no need to change that.

**Comment by RiccardoMasutti** [comment]
> Everything was working properly before the changes. there is no need to change that.

I saw that in my previous PRs you modified each .html link in .md

I checked all the documentation and modified all the links..

**Comment by dennisreimann** [comment]
We should use `.md` consistently – this way it will work either way: In the docs website and also when browsing the Markdown files on GitHub.

**Comment by dennisreimann** [comment]
I think we should close this one.

**Pull request merged**